wherobots/db/cursor.py

```diff
@@ -75,10 +75,16 @@ def fetchmany(self, size: int = None):
         return results
 
     def fetchall(self):
-        return self.__get_results()[self.__current_row :]
+        return self.__get_results()[self.__current_row:]
```

Comment:
Formatter wants the extra space; please keep it. I'll add the pre-commit stuff soon.